res/Sectors/M1105/Ricenden.txt

```diff
@@ -31,41 +31,44 @@
 
 # Alleg: Hf  : "Hive FDA Uplift/Observation Site"
 # Alleg: HvFd: "Hive Federation"
+# Alleg: Kr: "Krotan Primacy"
```

Comment:
The header for this sector says "Wrenton"

Reply:
Actually I seem to have duplicated Wrenton here, so I've got the last version back